Classes/Emogrifier.php

```diff
@@ -226,6 +226,13 @@ class Emogrifier
     ];
 
     /**
+     * If true, allow debug Exception to be thrown.
```

Comment:
Let's make this more specific: … Emogrifier will throw Exceptions when it encounters an error instead of silently ignoring them.